zetaclient/chains/sui/client/client.go

```diff
@@ -156,6 +156,38 @@ func (c *Client) GetOwnedObjectID(ctx context.Context, ownerAddress, structType
 	return res.Data[0].Data.ObjectId, nil
 }
 
+// SuiDevInspectTransactionBlock manual implementation of ISuiAPI.SuiDevInspectTransactionBlock
+// Don't use this function at this moment because Sui SDK currently returns deserialization error.
+//
+// @see sui.(*suiReadTransactionImpl).SuiDevInspectTransactionBlock
+// @see https://docs.sui.io/sui-api-ref#sui_devinspecttransactionblock
+func (c *Client) SuiDevInspectTransactionBlock(
```

Comment:
Let's simplify the naming. Also, do public RPCs / paid on-premise RPCs support this method?
```suggestion
func (c *Client) InspectTransactionBlock(
```

Reply:
same commit https://github.com/zeta-chain/node/pull/3765/commits/9def20d4f5cd946db3ecf002e7f62ddf64384d2b